cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -331,3 +335,31 @@ func withDeletionTimestamp(object ctrlclient.Object) {
 	now := metav1.Now()
 	object.SetDeletionTimestamp(&now)
 }
+
+type Table struct {
+	tableData []string
+}
+
+func MatchTable(expected []string) gtype.GomegaMatcher {
+	return &Table{tableData: expected}
+}
+
+func (t *Table) Match(actual interface{}) (bool, error) {
+	tableString := actual.(string)
+	tableParts := strings.Split(tableString, "\n")
+
+	for i := range t.tableData {
+		if !strings.HasPrefix(tableParts[i], t.tableData[i]) {
+			return false, fmt.Errorf("The Table does not match expected table output")
+		}
+	}
+	return true, nil
+}
+
+func (t *Table) FailureMessage(actual interface{}) string {
+	return fmt.Sprintf("Expected %v and received %v", actual.(string), t.tableData)
```

Comment:
can you please change the message here to have consistency between the expected and the actual values shown in this message. Consider making both of them either strings or both of them as slices.

Reply:
set to slices